client/extensions/woocommerce/state/ui/promotions/reducer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	WOOCOMMERCE_PROMOTIONS_PAGE_SET,
+} from 'woocommerce/state/action-types';
+
+const initialState = {
+	currentPage: 1,
+	perPage: 10,
```

Comment:
See question above about default for perPage (30 vs 10)

Reply:
Same as above. This is for display purposes, the 30 per page is for loading efficiently.